src/essreflectometry/amor/data.py

```diff
@@ -23,11 +26,14 @@ def _make_pooch():
 _pooch = _make_pooch()
 
 
-def get_path(name: str) -> str:
+def getpath(name: PoochFilename[Run]) -> FilePath[Run]:
```

Comment:
In ESSsans, these functions have this signature:
```python
def get_path(filename: FilenameType) -> FilePath[FilenameType]:
```
with
```python
FilenameType = TypeVar('FilenameType', bound=str)
class FilePath(sciline.Scope[FilenameType, str], str): ...
```

I don't know why this is parametrised. @nvaytet Can you clarify?

Reply:
Because `FilePath` can be used for any file, not just `Run` files, but also mask files, or the file containing the direct beam function. Does that answer your question?

Reply:
Yes, thanks!

So @jokasimr, can you change this to match the implementation in ESSsans except for the `FilenameType`?

Reply:
I can change it, but I'm not sure what you mean by matching the implementation in ESSsans except for `FilenameType`?

Should `FilePath` not be parameterized by anything or should it be parameterized by only the `RunType`?

Reply:
It's just about the names. Call the function `get_path` and the argument `Filename`. And keep the way it is parametrised.